geopyspark/tests/euclidean_distance_test.py

```diff
@@ -0,0 +1,69 @@
+import os
+import unittest
+import numpy as np
+
+import pytest
+
+from shapely.geometry import Point, MultiPoint, LineString
+from geopyspark.tests.base_test_class import BaseTestClass
+from geopyspark.geotrellis.rdd import TiledRasterRDD
+from geopyspark.geotrellis.constants import SPATIAL
+
+import pyproj
```

Comment:
pyproj will have to be added to `requirements.txt` if this is to pass on Travis. Either that, or an extra line will have to be added to the Travis installation script.